### PR TITLE
[PF-1743] For `terra resolve` data collection, print table

### PR DESCRIPTION
**Pull request by melissachang**
Before:

```
~/terra-workspace-manager (mc/pandas_gbq): terra resolve --name=1000-genomes
pedigree: bigquery-public-data.human_genome_variants.1000_genomes_pedigree
gcs-folder: gs://genomics-public-data/1000-genomes-phase-3
phase-3-variants: bigquery-public-data.human_genome_variants.1000_genomes_phase_3_variants_20150220
melissachang-terra-example-notebooks: git@github.com:melissachang/terra-example-notebooks.git
sample_info: bigquery-public-data.human_genome_variants.1000_genomes_sample_info
```

After:
```
~/terra-cli (mc/resolve): terra resolve --name=1000-genomes
NAME                                      PATH
pedigree                                  bigquery-public-data.human_genome_variants.1000_genomes_pedigree
gcs-folder                                gs://genomics-public-data/1000-genomes-phase-3
phase-3-variants                          bigquery-public-data.human_genome_variants.1000_genomes_phase_3_variants_20150220
melissachang-terra-example-notebooks      git@github.com:melissachang/terra-example-notebooks.git
sample_info                               bigquery-public-data.human_genome_variants.1000_genomes_sample_info
```

### CODE REVIEW

File: src/main/java/bio/terra/cli/command/resource/Resolve.java [hunk 1]
```diff
@@ -98,20 +103,71 @@ protected void execute() {
...
-    for (var resourceName : resourceNamesToCloudIds.keySet()) {
-      if (printResourceName) {
-        OUT.println(resourceName + ": " + resourceNamesToCloudIds.get((String) resourceName));
-      } else {
-        OUT.println(resourceNamesToCloudIds.get((String) resourceName));
-      }
+    // For a single resource, just print cloud ID (no resource name)
+    if (resourceNamesToCloudIds.length() == 1) {
+      String resourceName = (String) resourceNamesToCloudIds.keySet().iterator().next();
+      OUT.println(resourceNamesToCloudIds.get(resourceName));
```
Comment (zloery) [inline src/main/java/bio/terra/cli/command/resource/Resolve.java:109]: return here (or put the logic below inside `else` block). as-is this prints single resources twice, once with just the cloud ID then again as a table

**Review by zloery — APPROVED** [review]

File: src/main/java/bio/terra/cli/command/resource/Resolve.java (hunk 1 above)
Comment (melissachang) [reply src/main/java/bio/terra/cli/command/resource/Resolve.java:109]: Oops done, thanks

File: src/main/java/bio/terra/cli/command/resource/Resolve.java [hunk 2]
```diff
@@ -98,20 +103,72 @@ protected void execute() {
...
 
   private void printJson(JSONObject resourceNamesToCloudIds) {
     // "2" prevents entire dict from being printed on one line and to stay consistent with the rest
     // of JSON formatted output.
     OUT.println(resourceNamesToCloudIds.toString(2));
   }
+
+  private enum ResolveColumns implements ColumnDefinition<Pair<String, String>> {
+    NAME("NAME", Pair::getLeft, 40, Alignment.LEFT),
+    CLOUD_ID("PATH", Pair::getRight, 90, Alignment.LEFT);
```
Comment (yuhuyoyo) [inline src/main/java/bio/terra/cli/command/resource/Resolve.java:136]: rename the enum to PATH too?

File: src/main/java/bio/terra/cli/command/resource/Resolve.java [hunk 3]
```diff
@@ -98,20 +103,72 @@ protected void execute() {
...
-    for (var resourceName : resourceNamesToCloudIds.keySet()) {
-      if (printResourceName) {
-        OUT.println(resourceName + ": " + resourceNamesToCloudIds.get((String) resourceName));
-      } else {
-        OUT.println(resourceNamesToCloudIds.get((String) resourceName));
-      }
+    // For a single resource, just print cloud ID (no resource name)
+    if (resourceNamesToCloudIds.length() == 1) {
+      String resourceName = (String) resourceNamesToCloudIds.keySet().iterator().next();
+      OUT.println(resourceNamesToCloudIds.get(resourceName));
```
Comment (yuhuyoyo) [inline src/main/java/bio/terra/cli/command/resource/Resolve.java:109]: resourceNamesToPath?

**Review by yuhuyoyo — APPROVED** [review]

**Pull request merged**